.github/workflows/publish.yml

```diff
@@ -1,26 +1,38 @@
-name: Publish to Maven Central
+name: Publish to Maven Central and Dockerhub
 
 on:
   release:
-    types: [published]
+    types: [ published ]
 
 jobs:
-  build:
+  publish:
+    environment: mavenCentralPublish
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      - name: Checkout Extension SDK
+        run:
+          git clone https://github.com/hivemq/hivemq-extension-sdk.git ../hivemq-extension-sdk ||
+          cd ../hivemq-extension-sdk ||
+          git checkout ${GITHUB_REF##*/} || true ||
+          cd ../hivemq-community-edition
```

Comment:
maybe add a multiline command instead of the pipes (https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsrun)